simapp/simd/cmd/root.go

```diff
@@ -151,12 +150,9 @@ func txCommand() *cobra.Command {
 		authcmd.GetMultiSignCommand(),
 		authcmd.GetMultiSignBatchCmd(),
 		authcmd.GetValidateSignaturesCommand(),
-		flags.LineBreak,
 		authcmd.GetBroadcastCommand(),
 		authcmd.GetEncodeCommand(),
 		authcmd.GetDecodeCommand(),
-		flags.LineBreak,
-		vestingcli.GetTxCmd(),
```

Comment:
The vesting commands are already bundled with another module's CLI commands?

Reply:
`GetTxCmd` is called again via `AddTxCommands` on line 162.